plugins/org.eclipse.thym.ui.importer/src/org/eclipse/thym/ui/importer/Activator.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
```

Comment:
I think `Activator` is a horrible name. A workspace is typically full of it which makes code assist impossible